Microkelvin 0.9

krl opened the pull request:
fixes #151

krl committed:
Adapt to microkelvin 0.9

krl committed:
Adapt to microkelvin 0.8

ZER0 commented on CHANGELOG.md:
```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Changed the tree logic to be compatible with `microkelvin` 0.8
```
Present tense not past tense.
This is applying to the whole Changelog.

ZER0 commented on Cargo.toml:
```diff
@@ -18,7 +18,7 @@ dusk-bytes = "0.1"
 dusk-hades = "0.16.0-rc"
```
remove the rc, now hades is published

ZER0 commented on Cargo.toml:
```diff
@@ -18,7 +18,7 @@ dusk-bytes = "0.1"
 canonical = {version = "0.6", optional = true}
 canonical_derive = {version = "0.6", optional = true}
-microkelvin = {version = "0.7", optional = true}
+microkelvin = {version = "0.9.0-rc", optional = true}
```
just have `0.9` here

ZER0 commented on Cargo.toml:
```diff
@@ -18,7 +18,7 @@ dusk-bytes = "0.1"
 canonical_derive = {version = "0.6", optional = true}
-microkelvin = {version = "0.7", optional = true}
+microkelvin = {version = "0.9.0-rc", optional = true}
 nstack = {version = "0.8", optional = true}
```
Use `0.9` here.

ZER0 reviewed (approved):
With the comments addressed.

Commit:
Merge branch 'microkelvin_0.8' of https://github.com/dusk-network/Poseidon252 into microkelvin_0.8

Commit:
Add `persistance` feature to crate

- Update `microkelvin` to `v0.9.1`
- Update `nstack` to `0.9`
- Add `persistance` feature to the crate.

Commit:
Update CHANGELOG.md`

CPerezz committed:
Merge branch 'release-0.21.0' into microkelvin_0.8
